pkg/model/crd.go

```diff
@@ -68,6 +68,12 @@ type CRDField struct {
 	FieldConfig       *ackgenconfig.FieldConfig
 }
 
+//IsRequired return if the shape is marked as required in AWS SDK Private model
```

Comment:
nit: please place a space in between the `//` and the start of the docstring comment :)